app/models/select_team.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class SelectTeam < ApplicationRecord
+  def self.favorite(user)
+    team_id = user.favorite.team_id
+    Team.find(team_id)
```

Comment:
```suggestion
    user.favorite.team
```

こうではダメですかね？
そしてこれでOKであればメソッドを作る必要はないかも？

Reply:
`user.favorite.team`で大丈夫だったのでメソッドにするのをやめました!